tyxe/bnn.py

```diff
@@ -200,7 +200,7 @@ def fit(self, data_loader, optim, num_epochs, callback=None, num_particles=1, cl
         loss = TraceMeanField_ELBO(num_particles) if closed_form_kl else Trace_ELBO(num_particles)
         svi = SVI(self.model, self.guide, optim, loss=loss)
 
-        for i in range(num_epochs):
+        for i in trange(num_epochs):
```

Comment:
I wouldn't use `tqdm` by default, if someone wants a progress bar they can handle that through the callback.

Reply:
I added it as it is a pyro dependency, so should be on the users machine. But, yes can easily be removed. Maybe, a few common nice callbacks can be added to the library for convenience.